components/Avatar/avatar.css

```diff
@@ -0,0 +1,28 @@
+@import url(../../docs-css/reset.css);
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+
+body{
+    margin: 0;
+}
+
```

Comment:
As recommended by @aman-can , avoid using style directly on HTML tags. Also, you've already styled body in style.css